ios/ImagePickerManager.m

```diff
@@ -3,19 +3,20 @@
 #import <React/RCTConvert.h>
 #import <AVFoundation/AVFoundation.h>
 #import <Photos/Photos.h>
-#import <React/RCTUtils.h>
+#import <PhotosUI/PhotosUI.h>
 
 @import MobileCoreServices;
 
 @interface ImagePickerManager ()
 
-@property (nonatomic, strong) UIImagePickerController *picker;
```

Comment:
The presenting view controller retains the picker so there's no requirement to hold on to it